### ISSUE TITLE
SPI Bus

### ISSUE THREAD

**Issue by BluesBilly** [sstaub/Ethernet3#53]
Can you overload init() function? You can specify which SPI bus to use.

void EthernetClass::init(uint8_t sspin)
{
	W5100.setSS(sspin);
}

void EthernetClass::init(uint8_t sspin, SPIClass &spi)
{
	_spi = spi;
	W5100.setSS(sspin);
}

**Comment by sstaub** [comment]
Can you make a Pull Request? .init is reserved it should changed to .setSPI

**Comment by sstaub** [comment]
For change the CS pin there is `Ethernet.setCsPin(3); // set Pin 3 for CS`

**Comment by BluesBilly** [comment]
Indeed, setspi is more reasonable. You need to add the function of setspi instead of using global SPI.
